.github/workflows/pr-go-workspace-check.yml

```diff
@@ -32,7 +30,9 @@ jobs:
     - name: Check for go mod & workspace changes 
       run: |
         if ! git diff --exit-code --quiet; then
-          echo "Changes detected. Please run 'go work sync' and commit the changes."
+          echo "Changes detected:"
+          git diff
+          echo "Please run 'go work sync' and commit the changes."
```

Comment:
cc/ @toddtreece I was getting an error here but my local `go work sync` didn't cause any change. For some reason, in my local copy, some `indirect` annotations get removed :shrug: Added this to be able to debug issues.